components/List.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { nowPlayingMovies } from "../lib/db";
+import MovieCard from "./Card";
+
+const StyledList = styled.li`
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+export default function NowPlayinglist() {
```

Comment:
if you choose to give your files a name, it is better to give your components the same name.
And naming it `MovieList` comes closer to what is actually does

```suggestion
export default function MoviesList() {
```